PhysicsTools/PatAlgos/python/slimming/packedPFCandidates_cfi.py

```diff
@@ -12,5 +12,12 @@
       cms.InputTag("inclusiveCandidateSecondaryVertices"),
       cms.InputTag("inclusiveCandidateSecondaryVerticesCvsL"),
       ),      
-    minPtForTrackProperties = cms.double(0.95)
+    minPtForTrackProperties = cms.double(0.95),
+    covarianceVersion = cms.int32(0), #so far: 0 is Phase0, 1 is Phase1   
+#    covariancePackingSchemas = cms.vint32(1,257,513,769,0)  # a cheaper schema in kb/ev 
+    covariancePackingSchemas = cms.vint32(8,264,520,776,0)   # more accurate schema +0.6kb/ev   
```

Comment:
please add some notes what these values mean here.
[it looks like currently, the only solution is to run some script, read the code, and then match the numbers]